### PR TITLE
[nvccTests] Enable hipPeerToPeer_simple on nvcc

### PR DESCRIPTION
**Pull request by phani544**
Enable this on on nvcc. on hipcc it is crashing in a mGPU, non-peer scenario. Looks like that needs to be fixed.

### CODE REVIEW

File: tests/src/p2p/hipPeerToPeer_simple.cpp [hunk 1]
```diff
@@ -23,10 +23,10 @@ THE SOFTWARE.
 // Also serves as a template for other tests.
 
 /* HIT_START
- * BUILD: %t %s ../test_common.cpp
- * RUN: %t EXCLUDE_HIP_PLATFORM all
- * RUN: %t --memcpyWithPeer EXCLUDE_HIP_PLATFORM all
- * RUN: %t --mirrorPeers EXCLUDE_HIP_PLATFORM all
+ * BUILD: %t %s ../test_common.cpp 
+ * RUN: %t EXCLUDE_HIP_PLATFORM hipcc
```
Comment (mangupta) [inline tests/src/p2p/hipPeerToPeer_simple.cpp]: Not sure what you mean by ```EXCLUDE_HIP_PLATFORM hipcc```. Only valid values are ```hcc```, ```nvcc``` and ```all```.

Comment (mangupta) [reply tests/src/p2p/hipPeerToPeer_simple.cpp]: @phani544 can you update the code or respond to the above comment?

Comment (phani544) [reply tests/src/p2p/hipPeerToPeer_simple.cpp]: @mangupta, I will modify the test shortly.

Comment (mangupta) [reply tests/src/p2p/hipPeerToPeer_simple.cpp]: @phani544 could please push the updated code?

**Pull request merged**